README.md

```diff
@@ -1,2 +1,39 @@
 # sshd-docker
 Docker image with SSH daemon installed
+
+# Usage
+This container is meant to be used to expose host ports to containers running in Docker.
+When using testcontainers in unit tests, refer to the documentation of testcontainers to find out how to use it:
+- Java: https://java.testcontainers.org/features/networking/#exposing-host-ports-to-the-container
+- .NET: https://dotnet.testcontainers.org/api/create_docker_network/#exposing-host-ports-to-the-container
+- node.js: https://node.testcontainers.org/features/networking/#expose-host-ports-to-container
```

Comment:
Could you add Go too?
```suggestion
- node.js: https://node.testcontainers.org/features/networking/#expose-host-ports-to-container
- Go: https://golang.testcontainers.org/features/networking/#exposing-host-ports-to-the-container
```

Reply:
sounds good, I also tried finding it in haskell and rust but did not see it in the docs